specification/monitor/data-plane/readme.md

```diff
@@ -38,7 +38,7 @@ These settings apply only when `--tag=package-2018-09-preview` is specified on t
 
 ``` yaml $(tag) == 'package-2018-09-preview'
 input-file:
-- preview/2018-09-01-preview/azureMonitorCustomMetricsIngestionApi.json
+- preview/2018-09-01-preview/MetricsPost_API.json
```

Comment:
I'd remove the `_API` suffix from the file name, and if you think other operations will be added to this in the future perhaps a bit more generic of a name.

Reply:
Am not sure if there is any significance of the suffix. Currently we have Metrics_API that is to read metrics and the doc generation is working as expected. Hence we are copying the naming. Do you think it has no correlation to how doc generation works?

Reply:
No but the name isn't very nice.  Since you're changing the operation ID how about renaming to `MetricsCreate.json`